server/handlers/handlers.go

```diff
@@ -423,6 +430,10 @@ func (ah *ActivityHandler) handleCreatedActivity(msg *clientmodels.Message, subs
 }
 
 func (ah *ActivityHandler) handleUpdatedActivity(msg *clientmodels.Message, subscriptionID string, activityIds clientmodels.ActivityIds) string {
+	if ah.plugin.GetSyncNotifications() {
```

Comment:
nit: should we consider updating the quote in the bot message?

Reply:
Yes, we do need to define the behaviour here -- will track as a task.